spec/models/order_spec.rb

```diff
@@ -17,6 +17,31 @@
     expect(order).to be_finished
   end
 
+  describe 'scopes' do
+    let!(:open_order)     { create :order, state: 'open'     }
+    let!(:finished_order) { create :order, state: 'finished' }
+    let!(:received_order) { create :order, state: 'received' }
+    let!(:closed_order)   { create :order, state: 'closed'   }
+
+    it 'should retrieve open orders in the "open" scope' do
+      expect(Order.open.count).to eq(1)
+      expect(Order.open.first.id).to eq(open_order.id)
+    end
+
+    it 'should retrieve finished, received and closed orders in the "finished" scope' do
+      expect(Order.finished.count).to eq(3)
+      expect(Order.finished.first.id).to eq(finished_order.id)
+      expect(Order.finished.second.id).to eq(received_order.id)
+      expect(Order.finished.third.id).to eq(closed_order.id)
```

Comment:
depending on the order could cause a flacky test. checking if the item is included should be a better way to test that